staking/tls.go

```diff
@@ -176,15 +172,3 @@ func NewCertAndKeyBytesWithSecpKey(secpKey *secp256k1.PrivateKey) ([]byte, []byt
 	}
 	return certBuff.Bytes(), keyBuff.Bytes(), nil
 }
-
-func VerifyCertificate(cert *x509.Certificate) error {
```

Comment:
All this changes are just reverting diff that we have with avax. Probably was lost in previous merges.